test/nightly/flink_validation_fhir_high_availability_spec.rb

```diff
@@ -5,18 +5,18 @@
 require_relative '../env'
 
 # Prior to running this test:
-#   - run `./gradlew build publishToMavenLocal` in the project's root directory to build the hri-flink-validation-fhir jar
+#   - run `./gradlew build publishToMavenLocal` in the project's root directory to build the flink-validation-fhir jar
 #   - run `./gradlew copyNightlyTestDependencies` in the project's root directory to prep the jar for this test's use
 #
-# Anytime the hri-flink-validation-fhir jar is recompiled, the steps above need to be executed for these tests to pick up the
+# Anytime the flink-validation-fhir jar is recompiled, the steps above need to be executed for these tests to pick up the
 # new version
 
 describe 'Flink Validation FHIR High Availability Job' do
 
   before(:all) do
     TENANT_ID = 'test'
     BATCH_COMPLETION_DELAY = 5000
-    @branch_name = ENV['BRANCH_NAME']
+    @travis_branch = ENV['TRAVIS_BRANCH']
```

Comment:
Shouldn't all of these be `BRANCH_NAME`?

Reply:
Yes. They have now all been replaced.